style.css

```diff
@@ -620,13 +220,997 @@ hr {
+    padding-left: 0;
+    gap: 32px;
+  }
+
+  #skillset ul {
+    width: 100%;
+    font-family: 'Inter', sans-serif;
+    font-size: 22px;
+    line-height: 42.67px;
+    background-image: url("Assets/circulo.svg"), url("Assets/cuadro.svg"), url("Assets/diamond.svg");
+    background-repeat: no-repeat, no-repeat, no-repeat;
+    background-position: 26% 10.8%, 33% 78.1%, 20% 44.5%;
+  }
+
+  #contact h3 {
+    width: 327px;
+    height: 216px;
+    padding-left: 24px;
+    font-family: 'Inter', sans-serif;
+    font-size: 32px;
+    font-weight: 700;
+    line-height: 44px;
+    color: #172b4d;
+    text-align: center;
+  }
+
+  .h3 {
+    width: 164px;
+    height: 24px;
+    font-family: 'Inter', sans-serif;
+    font-weight: 700;
+    text-align: center;
+    font-size: 32px;
+    line-height: 44px;
+  }
+
+  #contact {
+    width: 375px;
+    height: 850px;
+    padding: 12px 0;
+    background: #ffff;
+    font-size: 17px;
+    font-weight: 400;
+    line-height: 24px;
+    align-items: center;
+    color: #ffff;
+  }
+
+  .flex-container {
+    display: flex;
+    align-items: center;
+    margin-left: 23px;
+  }
+
+  .form {
+    width: 327px;
+    margin-left: 0;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  .form-input {
+    width: 94%;
+    font-family: 'Inter', sans-serif;
+    font-weight: 400;
+    color: #979493;
+    border-color: #979493;
+    margin-top: 50px;
+    margin-left: 0;
+    padding: 5px 10px;
+    border-top: none;
+    border-left: none;
+    border-right: none;
+  }
+
+  .form-box {
+    width: 327px;
+    height: 90px;
+    font-family: 'Inter', sans-serif;
+    font-style: italic;
+    background: #fbf8f7;
+    margin: 24px 0;
+    border: none;
+  }
+
+  .send-button {
+    display: flex;
+    height: 48px;
+    padding-bottom: 78px;
+    justify-content: center;
+    width: 327px;
+  }
+
+  .social-media {
+    width: 375;
+    height: 40px;
+    padding: 98px;
+    padding-right: 98px;
+    padding-top: 17px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 20px;
+    background-image: url("Assets/Shape.svg");
+    background-repeat: no-repeat;
+    background-position-x: 50%;
+    background-position-y: 105px;
+  }
+
+  .profesional {
+    padding-left: 37px;
+    padding-top: 70px;
+    padding-bottom: 0;
+    margin-top: 0;
+    margin-bottom: 0;
+    height: 81px;
+    width: 290px;
+    font-family: 'Crete Round', serif;
+    font-size: 32px;
+    font-weight: 400;
+    line-height: 44px;
+    text-align: left;
+    color: white;
+  }
+
+  .daily {
+    height: 99px;
+    width: 295px;
+    margin-top: 0;
+    padding-left: 37px;
+    padding-top: 0;
+    font-family: 'Inter', sans-serif;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 26px;
+    letter-spacing: 0;
+    text-align: left;
+    color: #ffff;
+  }
 }
 
 @media only screen and (min-width: 768px) {
-
-} */
+  .boton {
+    display: none;
```

Comment:
- [ ] Your application is not responsive yet, @sinruidotec check the record below and see what looks your app
![6](https://github.com/sinruidotec/Portfolio-part-2-Microverse/assets/77843050/68b77476-8a32-40bd-9086-1cd0b2b51d79)